templates/web/common/includes/donate_banner.tt.html

```diff
@@ -106,56 +106,65 @@ <h3 class="donation-banner__main-title">[% f_lang("f_donation_title_year", { yea
       <p>[% lang("donation_hook_2024") %]</p>
     </div>
     <img src="/images/misc/donation-banners/donation-banner-group-photo.png" alt="group photo donation 2024" />
   </div>
   <div>
-    <div>
-      <div class="donation-banner-footer__main-section">
+    <div class="donation-banner__aside">
+      <div class="donation-banner__main-section">
         <img
           width="50"
           height="50"
           src="https://world.openfoodfacts.org/images/logos/logo-variants/CMJN-ICON_WHITE_BG_OFF.svg"
           alt="open food facts logo"
         />
-        <h3 class="donation-banner-footer__main-title">[% lang("donation_title_2024") %]</h3>
+        <h3 class="donation-banner__main-title">[% f_lang("f_donation_title_year", { year => "2025" } ) %]</h3>
+      </div>
+      <div style="padding:1rem;">
+        <p>[% lang("donation_list_2024_main") %]</p>
+        <ul>
+          <li>
+            [% lang("donation_list_2024_first") %]
+            <ul>
+              <li>[% lang("donation_list_2024_first_sub") %]</li>
+            </ul>
+          </li>
+          <li>
+            <p>[% lang("donation_list_2024_second") %]</p>
+          </li>
+          <li>
+            <p>[% lang("donation_list_2024_third") %]</p>
+          </li>
+          <li>
+            <p>[% lang("donation_list_2024_fourth") %]</p>
+          </li>
+        </ul>
       </div>
-      <p>[% lang("donation_list_2024_main") %]</p>
-      <ul>
-        <li>
-          [% lang("donation_list_2024_first") %]
-          <ul>
-            <li>[% lang("donation_list_2024_first_sub") %]</li>
-          </ul>
-        </li>
-        <li>
-          <p>[% lang("donation_list_2024_second") %]</p>
-        </li>
-        <li>
-          <p>[% lang("donation_list_2024_third") %]</p>
-        </li>
-        <li>
-          <p>[% lang("donation_list_2024_fourth") %]</p>
-        </li>
-      </ul>
     </div>
-    <div class="donation-banner-footer__actions-section">
-      <div class="donation-banner-footer__actions-section__financial">
+    <div class="donation-banner__actions-section">
+      <div class="donation-banner__actions-section__financial">
         <p>
+          [% IF cc == "fr" %]
+          (déductible à 66 % de vos impôts: ex. un don de 20€ vous reviendra à 6,80€)
+          [% ELSE %]
           [% lang("donation_financial_2024_text") %]
+          [% END %]
         </p>
       </div>
-      <div class="donation-banner-footer__actions-section__donate-button">
+      <div class="donation-banner__actions-section__donate-button">
         <a href="[% IF lc == "fr" %]https://open-food-facts.assoconnect.com/collect/description/476750-c-faire-un-don-a-open-food-facts[% ELSE %][% link %][% END %]?utm_source=off&utf_medium=web&utm_campaign=donate-2024-a&utm_term=en-text-button">
           <button>[% lang("donation_button_2024_text") %]</button>
         </a>
       </div>
     </div>
   </div>
+  <div class="donation-banner__close">
```

Comment:
Why are there changes to the donate_banner.tt.html template? It is unrelated to sessions.